.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDurationExtractor.cs

```diff
@@ -265,6 +250,9 @@ private List<ExtractResult> MergeMultipleDuration(string text, List<ExtractResul
                     continue;
                 }
 
+                // Add extraction to list of separate results (needed in case the extractions should not be merged)
+                List<ExtractResult> separaList = new List<ExtractResult>() { extractorResults[firstExtractionIndex] };

```

Comment:
This will have at most two elements, right? Couldn't this auxiliary list be replaced by two flags or a simple struct?

Reply:
no, in cases where there are 3 or more contiguous duration entities, they are all added to this list.